src/components/storage/CockpitStorageIntegration.scss

```diff
@@ -57,3 +60,25 @@ ul.cockpit-storage-integration-requirements-hint-list {
         color: unset;
     }
 }
+
+// Simulate Backdrop behavior for modals in the iframe
+%iframe-zindex {
+  position: relative;
+  z-index: 10;
+}
+
+iframe {
+  @extend %iframe-zindex;
+}
+
+// Simulate a PF background scrim
+.cockpit-has-modal {
+  @extend %iframe-zindex;
+
+  &::after {
+    @extend .pf-v5-c-backdrop;
```

Comment:
This `@extend` is different — it's using a PF class from CSS. (I think we're using precompiled CSS for PF? Either way, this is from some (S)CSS from PF which is known by SASS when this is getting compiled).

This looks like 

```scss
.pf-v5-c-backdrop, .cockpit-has-modal::after {
  /* PF specific code here */
}
```

So this means we're using exactly what PatternFly uses for the scrim, without having to duplicate the CSS and have drift if/when they change it. It will always be the exact same CSS as PF.